README.md

```diff
@@ -121,6 +124,7 @@ error when you try to launch your cluster. By default, the `aws_ami` property is
 You will need to changes this value if a newer image has been released or if you are running in different region
 than `us-east-1`.
 
+
```

Comment:
This can be a new sub-section, to differentiate it from the new "Configuring the AMI" sub-section above:

```suggestion
### Launching the cluster

```